src/Ocelot.Provider.Kubernetes/Ocelot.Provider.Kubernetes.csproj

```diff
@@ -25,8 +25,8 @@
     <NoWarn>1591</NoWarn>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="KubeClient" Version="2.5.12" />
-    <PackageReference Include="KubeClient.Extensions.DependencyInjection" Version="2.5.12" />
+    <PackageReference Include="KubeClient" Version="3.0.0" />
```

Comment:
Actually, it should be [3.0.1](https://www.nuget.org/packages/KubeClient/3.0.1) 😄

Reply:
Yeah, fair call - I'd only just published that; will pull in the new version 🙂

Reply:
Hmm - just before I push this change, I thought I'd better check - `KubeClient` `3.0.1` requires `Microsoft.Extensions.*` `9.0.3` (Ocelot's test projects currently use `9.0.1`;  tried upgrading, and all are tests still passing, but it's your call if you want me to upgrade).